remote/index.js

```diff
@@ -14,6 +14,8 @@ var express = require('express');
 var backstop = require('../core/runner');
 var { modifyJsonpReport } = require('../core/util/remote');
 
+const booleanizeArg = incrementalFlag => [true, 'true'].includes(incrementalFlag);
```

Comment:
What happens when incrementalFlag is `"false"`?

Reply:
This method(`booleanizeArg()`) will return boolean `false` when the input  is anything other than `true` or `'true'`.

Reply:
awesome.